### PR TITLE
DOC: Fixing 'a la' confusion in series.quantile documentation

### LINKED ISSUE #21292: DOC: doc string is not very clear for Series.quantile
#### Problem description
What is ``a la`` ? Is it French or a typo ? Suggest change it to something easy to understand 
https://github.com/pandas-dev/pandas/blob/f1631bec96dd9a1dc4890677b9c5475d0677e102/pandas/core/series.py#L1838-L1840

#### Output of ``pd.show_versions()``

<details>

INSTALLED VERSIONS
------------------
commit: None
python: 3.5.2.final.0
python-bits: 64
OS: Windows
OS-release: 10
machine: AMD64
processor: Intel64 Family 6 Model 94 Stepping 3, GenuineIntel
byteorder: little
LC_ALL: None
LANG: None

pandas: 0.18.1
nose: 1.3.7
pip: 9.0.1
setuptools: 27.2.0
Cython: 0.24.1
numpy: 1.13.3
scipy: 1.0.0
statsmodels: 0.6.1
xarray: None
IPython: 5.1.0
sphinx: 1.4.6
patsy: 0.4.1
dateutil: 2.6.1
pytz: 2017.3
blosc: None
bottleneck: 1.1.0
tables: 3.2.2
numexpr: 2.6.4
matplotlib: 2.1.1
openpyxl: 2.3.2
xlrd: 1.0.0
xlwt: 1.1.2
xlsxwriter: 0.9.3
lxml: 3.6.4
bs4: 4.5.1
html5lib: None
httplib2: None
apiclient: None
sqlalchemy: 1.0.13
pymysql: None
psycopg2: None
jinja2: 2.8
boto: 2.42.0
pandas_datareader: None

</details>

### PR DESCRIPTION
**Pull request by uds5501**
Just stating that numpy.percentile also offers something similar to this

- [x] closes #21292
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] Just clarified quantile doc a bit

### CODE REVIEW

File: pandas/core/series.py [hunk 1]
```diff
@@ -1837,7 +1837,8 @@ def round(self, decimals=0, *args, **kwargs):
 
     def quantile(self, q=0.5, interpolation='linear'):
         """
-        Return value at the given quantile, a la numpy.percentile.
+        Return value at the given quantile
+        (you can also see numpy.percentile, much similar like series.quantile).
```
Comment (WillAyd) [inline pandas/core/series.py]: Instead of this just add numpy.percentile to the See Also section

**Review by WillAyd — CHANGES_REQUESTED** [review]

**Comment by jorisvandenbossche** [comment]
@uds5501 general feedback: can you give the PR a more informative title? (you can still edit the title, see 'edit' button at the right of it)

**Comment by uds5501** [comment]
@WillAyd  Did the update you asked for. Now numpy.percentile is included in See also section

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
@uds5501 Thanks a lot!

**Pull request merged**